CODIGO/ModGameplayUI.bas

```diff
@@ -232,6 +233,29 @@ OnClick_Err:
     Resume Next
 End Sub
 
+Public Sub HandleQuestionResponse(ByVal Result As Boolean)
```

Comment:
Is this the callback that BABEL uses to notify the game the user choice?

Reply:
no, this is the common point for both babel and legacy ui, its called from both babel and here in HandleGameplayAreaMouseUp